README.md

```diff
@@ -64,8 +61,14 @@ to use.
 Installation
 ------------
 
-Flexx requires Python 3.5+ (only PyScript can be used with 2.7+) and also works on pypy. Further,
-it needs the [tornado](http://www.tornadoweb.org) library (pure Python).
+Flexx requires Python 3.5+ and also works on pypy. Further,
+it depends on:
+
+* the [tornado](http://www.tornadoweb.org) library (pure Python).
+* the [PScript](http://github.com/flexxui/webruntime) library (a flexxui project).
```

Comment:
These urls are incorrect

Reply:
Nice catch! I take it that you meticulously reviewed this whole PR? :P

Reply:
Nope TLDR